libs/s25main/controls/ctrlCheck.h

```diff
@@ -5,21 +5,34 @@
 #pragma once
 
 #include "Window.h"
+#include "ctrlBaseTooltip.h"
+#include <string>
 class MouseCoords;
 class glFont;
 
-class ctrlCheck : public Window
+class ctrlCheck : public Window, ctrlBaseTooltip
```

Comment:
is `ctrlBaseTooltip` used at all? nothing is marked override of the new stuff?

Reply:
It is used by the class similar to how it is used in others: On mouse move the tooltip is shown or hidden.

It could be a member instead though and likely should